config/jest/setup.js

```diff
@@ -6,6 +6,12 @@ jest.unmock('object-assign');
 
 require('../polyfills');
 
+const enzyme = require.requireActual('enzyme');
+const Adapter = require.requireActual('enzyme-adapter-react-15');
+
+// Configure enzyme to work with React 15 for now
+enzyme.configure({ adapter: new Adapter() });
```

Comment:
Is there a way to get a list of compatibility problems from Enzyme so we can make an issue to track remaining fixes?

Reply:
This is just for this PR, when we swap to React 16 it'll just use `enzyme-adapter-react-16` 😄